config/config.go

```diff
@@ -37,6 +37,11 @@ type Config struct {
 	// a user must pass via command line arguments.
 	AllowInsecureCreds bool `json:"-"`
 
+	// DisableProcesses is used to disable Processes for untrusted environments where
+	// a process cannot be trusted. This is an option a user must pass
+	// via command line arguments.
+	DisableProcesses bool `json:"-"`
```

Comment:
can we make this a broader untrusted flag? this should also disable the shell service as well and we can match on an untrusted env

Reply:
Yeah that makes sense. Though we should be careful about advertising too much trust at this point.

Reply:
What do you mean "and we can match on an untrusted env"

Reply:
oh as in just consulting the config and looking for untrusted to disable various features. bad wording :)

Reply:
Ah yeah gotcha. So far that's shell and processes.